webui/apps/eidolon-ui2/package.json

```diff
@@ -28,7 +28,7 @@
     "docker:down": "docker-compose down",
     "docker:build": "docker-compose build",
     "docker:rebuild": "pnpm run docker:down && pnpm run docker:build && pnpm run docker:up",
-    "test:e2e": "pnpm run docker:rebuild && pnpm exec playwright test && pnpm run docker:down",
+    "test:e2e": "pnpm exec playwright test",
```

Comment:
It's faster to develop tests if you start the container once and let it run. The only time you should need to `rebuild` is if you change the app code under test. Changing test code doesn't require a `rebuild`.

Reply:
Makes sense, and we can rebuild manually as needed.

It would be slick if we threw rebuild triggers on the docker compose. I have a similar example of this on the agent machine compose. Not blocking though.

Reply:
I found a resource that could help me learn how to do this. I'll give it a try in this PR 👍🏻

Reply:
No worries, I can give it a try in this PR. It'll be good for me as a Docker newbie 🤩

Reply:
> I have a similar example of this on the agent machine compose.

Where is this example, by the way? I've tried a few things so far and none of them work. I want to copy what was done for agent machine

Reply:
https://github.com/eidolon-ai/agent-machine/blob/38cdf40c2de241fab17cc8bb1cc59426d5abefc8/docker-compose.yml#L7